### ISSUE TITLE
design for the primary compute and io board

### ISSUE THREAD

**Issue by Raposa-Coltran** [Kelvin-Collaborations/Kelvin-1#1]
## getting the kelvin 1 off of the ground

this is the only unique piece of electronics manufacturing that should be nessecery for creating the kelvin 1, we should, i believe, start with a board designed for pi compute module double mezzanine form factors (see https://www.raspberrypi.com/products/compute-module-4/?variant=raspberry-pi-cm4001000 as the refference example of this formfactor), the board should provide:

- io breakout for the SBC
- io in the form of as many usbc connectors as is possible (prefferably usbc 4 instead of usbc 3) and no other io type

everything from bootable storage to wireless to screens to battery should be able to function fine over usbc, and this will make design and modification easier afterwards (discussion could be had about using a dedicated internal power cable for the board and battery for the sake of robustness, however,).

this needs to be done before anything else for this project can be seriously prototyped, the board must be designed and uploaded to the repository in an editable format and verifiably released under the project's mpl license ăs it is integral to the project), our next issue following this must discuss how we are to get prototypes for the board fabricated for testing.

**Comment by M-Edison** [comment]
Thankfully, after a night out with the lads, I saw a napkin blow out the saloon style bar doors and into the street.

Being a generally good person, I pick it up and find upon it sketched in eyeliner pencil the SLD for a Kelvin I I/O board. I was dumbfounded. I knew I had to share it here but the napkin smelled like a tequila sunrise already. Thankfully- I was able to transcribe it before this blessed wisdom faded away.

Picture Attached

    -Max

![image](https://github.com/Kelvin-Collaborations/Kelvin-1/assets/153699556/492b06a1-44fc-46e5-98d5-9e45aa7694f8)

**Comment by Raposa-Coltran** [comment]
the attatched story is fun, however your cursive in the diagram is kind of hard to parse

**Comment by Raposa-Coltran** [comment]
also, why is it in a tiered structure? i was moreso imagining doing _everyting_ via usb, screen and wifi included, so wouldnt it make more sense to attatch the compute module to the usb controller and run everything else off from it? in addition, i believe you forgot to add a connector for the battery power and fan controls (which prolly make sense to be combined together in terms of connection to the board)

**Comment by M-Edison** [comment]
Battery was shown, and it isn't tiered. This just shows some of the major component groups that'll be on the board and how they connect with one another. (PWR is the dotted line in the top)

**Comment by Raposa-Coltran** [comment]
ahh, yea i missed that, seems like it would be more elegant to just do everything over usbc tho

**Comment by M-Edison** [comment]
- All things that can be run off of USB4's protocol will be
- Some of these may have different names than "USB #n" because they will be dedicated use circuits. Don't be fooled, they are USB.

    - Max
